src/redux/books/books.js

```diff
@@ -1,44 +1,43 @@
+import axios from 'axios';
+
 const ADD_BOOK = 'book-store/books/ADD_BOOK';
 const REMOVE_BOOK = 'book-store/books/REMOVE_BOOK';
+const FETCH_BOOK = 'book-store/books/FETCH_BOOK';
+
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/5MiZYMR1tUdUdeapIDXc/books';
 
-const initialBook = [
-  {
-    id: 1,
-    author: 'Daniel Banda',
-    books: 'The fierce village',
-    category: '',
-  },
-  {
-    id: 2,
-    author: 'Gofrey Zimba',
-    books: 'Good deeds',
-    category: '',
-  },
-  {
-    id: 3,
-    author: 'Mathews Zulu',
-    books: 'The old village',
-    category: '',
-  },
-];
+const initialBook = [];
 
 const bookReducer = (state = initialBook, action) => {
   switch (action.type) {
     case ADD_BOOK: {
       const objBook = {
-        id: state.length + 1,
+        item_id: `book${Date.now()}`,
+        title: action.book.book,
         author: action.book.author,
-        books: action.book.books,
+        category: 'fiction',
       };
-      return [...state, objBook];
+      const newState = [`book${Date.now()}`, [{
+        title: action.book.book,
+        author: action.book.author,
+        category: 'fiction',
+      }]];
+
+      axios.post(baseUrl, objBook);
+      // console.log(state);
```

Comment:
- [ ] Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website, Kindly remove it 💯 👍🏻